pkgs/applications/misc/mkgmap/splitter/default.nix

```diff
@@ -74,7 +95,7 @@ stdenv.mkDerivation rec {
     downloadPage = "https://www.mkgmap.org.uk/download/splitter.html";
     sourceProvenance = with sourceTypes; [
       fromSource
-      binaryBytecode  # deps
+      binaryBytecode # deps
     ];
     license = licenses.gpl2Only;
     maintainers = with maintainers; [ sikmir ];
```

Comment:
```suggestion
    maintainers = with maintainers; [ sikmir ];
    mainProgram = "splitter";
```

Reply:
I added `mainProgram` and reordered the attributes asciibetically. If you don't want that, I can revert.